rosette/build.sh

```diff
@@ -23,7 +23,7 @@ if [ ! -d ${BUILD} ] ; then {
 } ; fi
 
 cd ${BUILD}
-cmake -B${BUILD} -H${SRCDIR}
+cmake -D CMAKE_CXX_COMPILER="/usr/bin/g++" -B${BUILD} -H${SRCDIR}cmake -D CMAKE_CXX_COMPILER="/usr/bin/g++" -B${BUILD} -H${SRCDIR}
```

Comment:
Wait... I think you duplicated the same code twice here. Can you fix before you merge?

Reply:
Oh, sorry.